conans/client/graph/graph_binaries.py

```diff
@@ -114,8 +131,11 @@ def _evaluate_node(self, node, build_mode, update, evaluated_references, remote_
                 else:
                     output.warn("Can't update, no remote defined")
             if not node.binary:
-                node.binary = BINARY_CACHE
-                package_hash = ConanInfo.load_from_package(package_folder).recipe_hash
+                if node.recipe == RECIPE_EDITABLE:
```

Comment:
Same comment here. I know I'm missing something... I would need some live explanation of this PR

Reply:
Me too... I can't remember now.